controllers/configmap_controller.go

```diff
@@ -340,3 +340,51 @@ func (r *ConfigMapReconciler) createServicesConfigMap(ctx context.Context, names
 	}
 	return windowsServices, nil
 }
+
+// EnsureServicesConfigMapExists ensures that the ServicesConfigMap is present and valid on the cluster during operator
+// bootup. ConfigMapReconciler.createServicesConfigMap() cannot be called in its stead as the cache has not been
+// populated yet, which is why the typed client is used here as it calls the API server directly.
+func (r *ConfigMapReconciler) EnsureServicesConfigMapExists() error {
+	var windowsServices *core.ConfigMap
+	var err error
+	if windowsServices, err = r.k8sclientset.CoreV1().ConfigMaps(r.watchNamespace).Get(context.TODO(),
```

Comment:
This should be turned into
```
windowsServices, err := r.k8sclientset.CoreV1().ConfigMaps(r.watchNamespace).Get(context.TODO(),
		servicescm.ServicesConfigMapName, meta.GetOptions{})
if err != nil {
...
```

as the types for windowsService and err don't need to be specified